README.md

```diff
@@ -16,15 +16,26 @@ Authors  :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano (caggiano@gma
 
 `MyoSuite` is a collection of musculoskeletal environments and tasks simulated with the [MuJoCo](http://www.mujoco.org/) physics engine and wrapped in the OpenAI ``gym`` API to enable the application of Machine Learning to bio-mechanic control problems.
 
- [Full task details](https://github.com/myohub/myosuite/blob/main/docs/source/suite.rst#tasks) | [Baselines](https://github.com/myohub/myosuite/tree/main/myosuite/agents/baslines_NPG) | [Documentation](https://myosuite.readthedocs.io/en/latest/)
-| [Tutorials](https://github.com/myohub/myosuite/tree/main/docs/source/tutorials)
+
+ <!-- TODO: add brieft history of the mujoco environment? -->
+<!-- TODO: the tutorials section needs to format as now it is mixing all versions -->
+ <!-- [Full task details](https://github.com/myohub/myosuite/blob/main/docs/source/suite.rst#tasks) | [Baselines](https://github.com/myohub/myosuite/tree/main/myosuite/agents/baslines_NPG) | [Documentation](https://myosuite.readthedocs.io/en/latest/)
+| [Tutorials](https://github.com/myohub/myosuite/tree/main/docs/source/tutorials) -->
+
+[Documentation](https://myosuite.readthedocs.io/en/latest/) | [Tutorials](https://github.com/myohub/myosuite/tree/main/docs/source/tutorials) | [Task specifications](https://github.com/myohub/myosuite/blob/main/docs/source/suite.rst#tasks)
+
 
 Below is an overview of the tasks in the MyoSuite.
 
 <img width="1240" alt="TasksALL" src="https://github.com/myohub/myosuite/blob/main/docs/source/images/myoSuite_All.png?raw=true">
 
 
-## Getting Started
+<!-- TODO: Section is important but seemed to me as an additional notes -->
```

Comment:
@Vittorio-Caggiano @raku-slyu

Reply:
This part is renamed and TODO msg removed on new PR